.circleci/config.yml

```diff
@@ -255,14 +255,15 @@ jobs:
               pip3 install -r .circleci/build-requirements.txt >> /tmp/lint_env_installation.log
               
               # installing local sdk inside venv
+              pip3 uninstall -y demisto-sdk
               pip3 install ~/project >> /tmp/lint_env_installation.log
```

Comment:
remove this line and run with poetry run.

Reply:
No, we install `demisto-sdk` to the local env (in line 259). It's installed before in `bootstrap`, and afterwards `demisto-sdk` refers to the new `demisto-sdk` installed.